Dockerfile.server

```diff
@@ -5,6 +5,21 @@ FROM node:5.5
 
 USER root
 
+# Install Xvfb and its dependencies needed to run Electron (Nightmare tests)
+RUN apt-get update && \
+    apt-get install -y \
+    libgtk2.0-0 \
+    libgconf-2-4 \
+    libasound2 \
```

Comment:
The list is fine. You done good. I'm commenting on the state of docker/testing/package management/how-much-does-disk-cost-again? affairs.